acrobat/scripts/scripts.js

```diff
@@ -287,6 +287,7 @@ const CONFIG = {
     /www\.adobe\.com\/(\w\w(_\w\w)?\/)?express(\/.*)?/,
     /www\.adobe\.com\/(\w\w(_\w\w)?\/)?go(\/.*)?/,
   ],
+  imsScope: 'AdobeID,openid,gnav,pps.read,firefly_api,additional_info.roles,read_organizations',
```

Comment:
If you want to take this to Production soon, make sure that the client ID has the additional scopes in IMSS as well. I have only added them to STAGE, waiting for FEDS team to test and provide a sign off before adding the scopes to PROD.

Reply:
@narcis-radu just updated the IMSS config. Didn't want to on Friday. 
This code will go live tomorrow around 10am PST. Thank you